vpr/src/server/telegrambuffer.h

```diff
@@ -20,11 +20,16 @@ namespace comm {
 */
```

Comment:
return -> returns
A bit more context on what the purpose of a telegram buffer is and what a "well filled frame" is would be helpful here.

Reply:
solved 56d8abc2f6389379b9674abe388914db063620b1